.idea/.generators

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
ewww. get these files outa here. They're specific to you / your machine so they shouldn't be in a shared repository.